plugins/contactpressuresensorsarray/include/yarp/dev/ContactPressureSensorsArrayDriver.h

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#ifndef GAZEBOYARP_CONTACTPRESSURESENSORSARRAYDRIVER_H
+#define GAZEBOYARP_CONTACTPRESSURESENSORSARRAYDRIVER_H
+
+#include <yarp/dev/DeviceDriver.h>
+#include <yarp/dev/IAnalogSensor.h>
+#include <yarp/dev/PreciselyTimed.h>
+#include <yarp/os/Stamp.h>
+#include <yarp/os/Semaphore.h>
+
+#include <yarp/sig/Matrix.h>
+
+#include <gazebo/common/Plugin.hh>
+
+#include <gazebo/sensors/Sensor.hh>
+#include <gazebo/sensors/ContactSensor.hh>
+#include <gazebo/sensors/SensorManager.hh>
+
+#include <gazebo/physics/Contact.hh>
+#include <gazebo/physics/Link.hh>
+#include <gazebo/physics/physics.hh>
+
+#include <ignition/math/Pose3.hh>
+#include <ignition/math/Vector3.hh>
+#include <ignition/math/Matrix3.hh>
+#include <ignition/math.hh>
+
+namespace yarp
+{
+    namespace dev
+    {
+        class GazeboYarpContactPressureSensorsArrayDriver;
+    }
+}
+
+/**
+ * YARP Device Driver for an array of contact pressure sensors
+ * 
+ * This device is used to simulate the contact normal forces that might be 
+ * observed by an arbitrary number of load cells at specific locations with respect
+ * to a given link's origin frame. It uses the instance of a Gazebo Contact Sensor which
+ * provides contact information of the link (specifically providing wrenches acting at the CG). 
+ * This information is transformed and mapped to the desired contact normal forces.
+ * 
+ * In its current version, this device reads the location of the load cells (to be mentioned manually from the urdf or sdf)
```

Comment:
"to be mentioned" ?

Reply:
copy the load cell locations from the urdf/sdf into the config file ?

Reply:
If you write the full sentence you want to add in the documentation it is easier for me to comment on it.